src/routes/LandingPage/EventsBanner/EventsBanner.jsx

```diff
@@ -0,0 +1,64 @@
+/* eslint-disable */
+
+import React from 'react';
+import './index.scss';
+import classnames from 'classnames'
+
+const EventsBanner = () => {
+  return (
+    <div className='evts-banner'>
+      <EventsDivider highlight='1' />
+      <EventsDescription text='Lorem ipsum dolor sit amet, consectetur adipiscing elit, sed do eiusmod tempor incididunt ut labore et dolore magna aliqua. Ut enim ad minim veniam, quis nostrud exercitation ullamco laboris nisi ut aliquip ex ea commodo consequat.
+        Duis aute irure dolor in reprehenderit in voluptate velit esse cillum dolore eu fugiat nulla pariatur.'/>
+      <EventsDivider highlight='6' />
+    </div>
+  );
+};
+//
+
+const EventsDescription = ({ text }) => {
+  return (
+    <div className='evts-desc'>
+      <p1>
+        {text}
```

Comment:
Just something to consider for next sprint, would be to import `Typography` from `antd` and then use the `Text` object from that. You can have a look at the footerbar to see how it may be used and also their documentation:
https://ant.design/components/typography/

Don't worry about that before merging this in though, feel free to resolve thsi conversation as soon as you see it.

Reply:
in an ideal world where this repo is in typescript, this would be less of an issue 😔